app/controllers/concerns/api/v2/taxonomies_controller.rb

```diff
@@ -20,6 +21,7 @@ module Api::V2::TaxonomiesController
       param :compute_resource_ids, Array, N_("Compute resource IDs"), :required => false
       param :media_ids, Array, N_("Media IDs"), :required => false
       param :config_template_ids, Array, N_("Provisioning template IDs"), :required => false
+      param :provisioning_template_ids, Array, N_("Provisioning template IDs"), :required => false
```

Comment:
I was against custom deprecated string here, it's planned to have support for marking params as deprecated in apipie, maybe leaving a comment to easily find this later would be enough